rust_src/src/buffers.rs

```diff
@@ -24,10 +24,12 @@ use crate::{
     lists,
     lists::{car, cdr, list, member, rassq, setcar},
     lists::{CarIter, LispCons, LispConsCircularChecks, LispConsEndChecks, TailsIter},
+    marker::set_marker,
```

Comment:
Why is it not grouped with the other items from `marker`?

Reply:
I will put it with the others; I noticed that, e.g. the imports from list were separated, and theorized that the idea was that each PR would separate the imports to not conflict, but as I write that, I realize it doesn't really make sense. In any case, I'll change that.